agentai/conversation.py

```diff
@@ -11,7 +13,7 @@ class Message(BaseModel):
 
 
 class Conversation:
-    def __init__(self, history: List[Message] = [], id: Optional[str] = None):
+    def __init__(self, history: List[Message] = [], id: Optional[str] = None, max_tokens: int = 100):
```

Comment:
```suggestion
    def __init__(self, history: List[Message] = [], id: Optional[str] = None, max_history_tokens: int = 200):
```

Reply:
I removed max_history_length and just kept max_tokens. max_history_length should do what?

Reply:
This is the maximum number of tokens we want to keep — not messages. And these are history tokens, not the max_tokens we might use in the agent prompt or future conversation. This variable names makes both of these things explicit. 

I've made the changes needed on `main` directly and tested the code. Consider this done from your end and pick up #6